packages/datepickers/src/styled/StyledMenu.tsx

```diff
@@ -93,53 +123,51 @@ StyledMenuView.defaultProps = {
 };
 
 interface IStyledMenuWrapperProps {
+  isHidden?: boolean;
+  isAnimated?: boolean;
+  zIndex?: number;
   placement?: POPPER_PLACEMENT;
 }
 
 const StyledMenuWrapper = styled.div<IStyledMenuWrapperProps>`
-  ${retrieveMenuMargin};
+  transition: ${props =>
+    props.isHidden && props.isAnimated && 'opacity .2s ease-in-out, .2s visibility 0s linear'};
+  visibility: ${props => props.isHidden && 'hidden'};
+  opacity: ${props => props.isHidden && '0'};
+  z-index: ${props => props.zIndex};
+  font-size: 0.01px; /* [1] */
```

Comment:
I noticed that `[1]` reference doesn't point to a description. Should there be a description for this `[1]`?

Reply:
Looks like I lost some comments during the refactor. I've added this styling comment back in.